### ISSUE TITLE
Unclear exception message when domains can't be validated

### ISSUE THREAD

**Issue by flan** [will-in-wi/letsencrypt-webfaction#40]
Message: Failed to verify statuses in 10 seconds. (RuntimeError)

Looking through the issue history, this has confused people a few times before. It's technically correct, but not really informative, in that it doesn't say why the failure occurred.

From the code, the actual check routine is in the acme-client logic, but printing a message like the following would probably help to reduce noise and give users enough information to solve the issue:

Failed to verify statuses in 10 seconds. (RuntimeError)
Verify that you can reach each of the following URLs:
- http://domain/.well-known/acme-challenge/x
- ...

**Comment by will-in-wi** [comment]
This is a great idea. In the next release, the output in such a case will look like:

```
Failed to verify statuses.
domain: DNS problem: NXDOMAIN looking up A for domain
Make sure that you can access http://domain/.well-known/acme-challenge/x
```

**Comment by will-in-wi** [comment]
This has been fixed in version 1.1.7, just released.
